custom_components/rfplayer/__init__.py

```diff
@@ -278,31 +288,28 @@ class RfplayerDevice(RestoreEntity):
     # pylint: disable-next=too-many-arguments
     def __init__(
         self,
+        unique_id: str,
```

Comment:
maybe rename in event_id as it's the variable named used in each call ?

Reply:
it's coming as event_id and propagated as unique_id to hass so both works. but I can change if you prefer.

Reply:
@Aohzan I guess you were confused by @racletteparty moving the position of this argument. Ie this argument was already there and named unique_id. It only moved place in the function signature.